### ISSUE TITLE
[Accessibility][Label and Instruction - Property Pane_Ask a question (Bot Asks)]: Ensures every form element has a label

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2140]
**User Experience** 
Users who rely on AT will face difficulty to locate the focus if the alt attribute is not defined for the image.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. "Bot Framework Composer" homepage will gets opened.
3. Select any of the previously created bot from "Recent Bots" or create one from the available Bots under 'Examples' and activate it.
4. Navigate to already created dialog tree item and activate it. Such as- Greetings, Actions, EndDialog etc.
5. Click on "+" and select 'Ask a question'. Then select any of the available options except Oauth login.
6. Navigate to property pane appeared at right side of the screen and select 'Bot Asks'.
7. Run Accessibility insight and observe the issue.

**Actual:** 
Bot Framework logo image doesn't have alternate text or a role of none or presentation.

**Expected:**
Ensures Bot Framework logo image have alternate text or a role of none or presentation.

**Note:**
This accessibility issue was found using Accessibility Insights for Web 2.14.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Observation:**
Same issue observe throughout the website for every property pane.

**Element path:**
.css-vaviek;#TextField942

**Snippet:**
`<input type="text" id="TextField942" placeholder="ActionsSample-0111" autocomplete="off" class="ms-TextField-field field-157" aria-invalid="false" value="ActionsSample-0111">`

**How to fix:**
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty

**MAS Impacted:** [MAS3.3.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf})

**Attachment:**
![Element must have label](https://user-images.githubusercontent.com/60913131/75650123-0ef6a780-5c7b-11ea-8de9-9a473b54138d.png)
![Element must have labelother](https://user-images.githubusercontent.com/60913131/75650376-aeb43580-5c7b-11ea-974a-70e86fe95374.png)

**Comment by ashish315** [comment]
@beyackle As checked on below environment the issue is still repro.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
Browser: Microsoft Edge Canary {Version 83.0.474.0 (Official build) canary (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
